modules/product/product.model.js

```diff
@@ -55,12 +55,20 @@ const productSchema = new mongoose.Schema({
   purchasedCount: Number,
   rate: Number,
   rateCount: Number,
-  votedUsers: [
-    {
+  userRates: [{
+    user: {
       type: mongoose.Schema.Types.ObjectId,
       ref: 'User',
     },
-  ],
+    rate: {
+      type: Number,
+      default: 0,
```

Comment:
take out the default value, the user should decide it.

Reply:
Done